src/components/TokenBalance.tsx

```diff
@@ -119,13 +120,35 @@ function useErrorMessageWithRefresh() {
   }, [shouldShowError])
 }
 
-export function AssetsTokenBalance() {
+export function AssetsTokenBalance({ showInfo }: { showInfo: boolean }) {
   const { t } = useTranslation()
+  const [infoVisible, setInfoVisible] = useState(false)
+
+  const toggleInfoVisible = () => {
+    setInfoVisible((prev) => !prev)
+  }
 
   return (
     <View testID="AssetsTokenBalance">
-      <Text style={styles.totalAssets}>{t('totalAssets')}</Text>
+      <View style={styles.row}>
+        <Text style={styles.totalAssets}>{t('totalAssets')}</Text>
+        {showInfo && (
+          <TouchableOpacity
+            onPress={toggleInfoVisible}
```

Comment:
I think it's a bit weird to dismiss the message only when tapping again in the info icon. Didn't you feel it weird?
I would expect that tapping anywhere would dismiss it.

Reply:
yeah this is a good point, it's not trivial to enable this on tap of _everywhere_ though so for now i've just added a handler to dismiss on tap of the header component